index.js

```diff
@@ -167,7 +169,8 @@ const meow = (helpText, options = {}) => {
 
 	normalizePackageData(package_);
 
-	process.title = package_.bin ? Object.keys(package_.bin)[0] : package_.name;
+	// eslint-disable-next-line node/prefer-global/process
+	global.process.title = package_.bin ? Object.keys(package_.bin)[0] : package_.name;
```

Comment:
Why not use the imported `process`?

Reply:
I faced some issues with `ava@4`. This is my try-out code and I forgot to change it back.